rclcpp/include/rclcpp/node_interfaces/node_base.hpp

```diff
@@ -98,6 +99,14 @@ class NodeBase : public NodeBaseInterface, public std::enable_shared_from_this<N
     rclcpp::CallbackGroupType group_type,
     bool automatically_add_to_executor_with_node = true) override;
 
+  RCLCPP_PUBLIC
+  rclcpp::CallbackGroup::SharedPtr
+  get_builtin_performance_callback_group() override;
+
+  RCLCPP_PUBLIC
+  rclcpp::CallbackGroup::SharedPtr
+  get_builtin_callback_group() override;
+
```

Comment:
Can we use the `NodeBase::create_callback_group` directly if calling these two methods somewhere?

Reply:
Do you mane rename them as "create_" prefix ?

Reply:
After the F2F discussion, we keep these 2 interfaces.